languages.toml

```diff
@@ -408,11 +408,23 @@ indent = { tab-width = 2, unit = "  " }
 name = "scss"
 source = { git = "https://github.com/serenadeai/tree-sitter-scss", rev = "c478c6868648eff49eb04a4df90d703dc45b312a" }
 
+[[language]]
+name = "htmldjango"
+scope = "source.htmldjango"
+injection-regex = "htmldjango"
+file-types = ["**/templates/**.html"]
+roots = ["manage.py"]
+indent = { tab-width = 2, unit = "  " }
+
+[[grammar]]
+name = "htmldjango"
+source = { git = "https://github.com/interdependence/tree-sitter-htmldjango", rev = "184a50456186c2ff49b9b410f7060a176e2a3080" }
+
 [[language]]
 name = "html"
 scope = "text.html.basic"
 injection-regex = "html"
-file-types = ["html"]
+file-types = ["*.html"]
```

Comment:
One of the downsides of introducing globbing is that it becomes ambiguous whether all `file-type`s should be `*.extension` or just `extension`. In this case, this has to be `*.html` so the globs for htmldjango and html are both checked, right? I'm worried that behavior is too nuanced and gets confusing

I think it would be more predictable to have `language_config_for_file_name` allow ties and then decide between the ties based on any `roots` as you were saying above (https://github.com/helix-editor/helix/pull/3249#discussion_r934109113)

Reply:
I added hueristic and won't rely on glob now.